ObjectCreation/AeroFoil.FCMacro

```diff
@@ -0,0 +1,1747 @@
+#!/usr/bin/env python
```

Comment:
Thank you Mr. Galou. 
This macro incorporates 3 sets of completely different ways of creating an airfoil; hence the very long file. 
I would be prepared to do split the file. That's an excellent idea. Meaning, I would have to add a set of well defined commands. That would take a few days.
Right now, as per the forum, there have been a couple errors I've just solved; also I'm planning to another important feature to the code (splitting the airfoil curve in two) as requested by a user.
Hence, I've closed the pull request. I'll get that done, and then submit a brand new v1.0.2 pull request.

Reply:
Even though the file is long, it's functionality is small; hence it cannot be a workbench. Although, sometime in the future, it can be part of an "Aerospace" workbench that would contain this macro along with others; someone had created a rocketry workbench, that could come under the greater Aerospace workbench too.

Reply:
I already integrated the code, just write another PR and I'll integrate it.

Reply:
Actually, I though that I had already integrated it. No problem to postpone integration.

Reply:
> Can you please modify the Wiki page to state that the macro can be installed via the official Add-ons Manager? Thanks.

I'll do that right away.

Reply:
> Actually, I though that I had already integrated it. No problem to postpone integration.

I didn't want to submit a buggy code. I'll send in another pull request within a couple hours.

Reply:
> This is a very long file, i.e. hard to maintain. Would it be possible to split it into a simple `*.FCMacro` and the core, which I would put a directory called `aero_foil`. There are two other advantages:
> 
>     * the core algorithms can be called from other macros or the Python console.
> 
>     * this is a condition to have the created object be parametric, even after FreeCAD restart (more on that below).
> 
> 
> I would then put the ui files in `aero_foil/resource`.

I'll be working on that for the future version.
Moreover, I had also [put up a question](https://forum.freecadweb.org/viewtopic.php?f=10&t=56431) asking if predefined tmp, src, ui folders could be created for the FC Macros. It could be a convention to easily store and obtain data across macros.